include/verilated_replay_common.h

```diff
@@ -0,0 +1,61 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Copyright 2003-2020 by Todd Strader. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License.
+// Version 2.0.
+//
+// Verilator is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+//=========================================================================
+///
+/// \file
+/// \brief Verilator: Include for common replay code
+///
+///     See verilator_replay
+///
+/// Code available from: http://www.veripool.org/verilator
+///
+//=========================================================================
+
+
+#ifndef _VERILATED_REPLAY_COMMON_H_
+#define _VERILATED_REPLAY_COMMON_H_ 1  ///< Header Guard
+
+#include "gtkwave/fstapi.h"
+#include <string>
+#include <map>
+#include <unordered_set>
+
+class VerilatedReplayCommon {
```

Comment:
This class is currently used by both verilator_replay and the Verilator replay runtime.  Is there any other code which is shared between the compiled tool and the Verilator runtime?  I don't believe there is.

Suggestions on where to put this?  I suspect this isn't the best way to do it.